tools/a-star/src/Main.java

```diff
@@ -0,0 +1,95 @@
+import astar.Spot;
+import drawing.DrawingBoard;
+import drawing.MyDrawable;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+
+import static astar.A_Star.A_Star;
```

Comment:
Inconsistent package naming, according to oracle guidelines, no underscores/caps in Package names

Reply:
package name is lowercase, A_Star is the name of the Class, but I do agree the name of the method should be lowercase